<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EmployeeInterface</class>
 <widget class="QDialog" name="EmployeeInterface">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1185</width>
    <height>605</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>CARSYS RENTALS</string>
  </property>
  <widget class="QLabel" name="btnRent">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>30</y>
     <width>275</width>
     <height>45</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI</family>
     <pointsize>20</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color:white;</string>
   </property>
   <property name="text">
    <string>CARSYS RENTALS</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblBackgroundTitle">
   <property name="geometry">
    <rect>
     <x>11</x>
     <y>20</y>
     <width>1163</width>
     <height>71</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>img/background1.jpg</pixmap>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>11</x>
     <y>11</y>
     <width>16</width>
     <height>16</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout"/>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>540</y>
     <width>521</width>
     <height>51</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_4">
    <item>
     <widget class="QPushButton" name="backToAdminBtn">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton#backToAdminBtn {
background-color: #6246ff; 
  border: none;
  color: white;
  padding: 15px 32px;
  text-align: center;
  text-decoration: none;
  display: inline-block;
  font-size: 16px;
  transition-duration: 0.4s;
}

QPushButton#backToAdminBtn:hover {
  background-color: #215a91; /* Green */
  color: white;
}</string>
      </property>
      <property name="text">
       <string>Go back to Admin</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="backToLoginBtn">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton#backToLoginBtn {
background-color: #6246ff; 
  border: none;
  color: white;
  padding: 15px 32px;
  text-align: center;
  text-decoration: none;
  display: inline-block;
  font-size: 16px;
  transition-duration: 0.4s;
}

QPushButton#backToLoginBtn:hover {
  background-color: #215a91; /* Green */
  color: white;
}</string>
      </property>
      <property name="text">
       <string>back to Login</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btnEmplQuit">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton#btnEmplQuit{
background-color: #ff5500; 
  border: none;
  color: white;
  padding: 15px 32px;
  text-align: center;
  text-decoration: none;
  display: inline-block;
  font-size: 16px;
  transition-duration: 0.4s;
}

QPushButton#btnEmplQuit:hover {
  background-color: #ffaa00; /* Green */
  color: white;
}</string>
      </property>
      <property name="text">
       <string>Quit</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>98</y>
     <width>941</width>
     <height>431</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="title">
    <string>Available Vehicles</string>
   </property>
   <widget class="QTableView" name="availableVehTable">
    <property name="geometry">
     <rect>
      <x>14</x>
      <y>40</y>
      <width>941</width>
      <height>321</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="refreshBtn">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>370</y>
      <width>49</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="printBtn">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>370</y>
      <width>49</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="updateBtn">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>370</y>
      <width>49</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>150</y>
     <width>201</width>
     <height>326</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QPushButton" name="btnRent_2">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <pointsize>-1</pointsize>
        <underline>false</underline>
        <strikeout>false</strikeout>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton#btnRent_2 {
background-color: #6246ff; 
  border: none;
  color: white;
  padding: 15px 32px;
  text-align: center;
  text-decoration: none;
  display: inline-block;
  font-size: 16px;
  transition-duration: 0.4s;
}

QPushButton#btnRent_2:hover {
  background-color: #215a91; /* Green */
  color: white;
}</string>
      </property>
      <property name="text">
       <string>Rent</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btnReturn">
      <property name="font">
       <font>
        <pointsize>-1</pointsize>
        <underline>false</underline>
        <strikeout>false</strikeout>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton#btnReturn {
background-color: #6246ff; 
  border: none;
  color: white;
  padding: 15px 32px;
  text-align: center;
  text-decoration: none;
  display: inline-block;
  font-size: 16px;
  transition-duration: 0.4s;
}

QPushButton#btnReturn:hover {
  background-color: #215a91; /* Green */
  color: white;
}</string>
      </property>
      <property name="text">
       <string>Return</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btnAddVehicle">
      <property name="font">
       <font>
        <pointsize>-1</pointsize>
        <underline>false</underline>
        <strikeout>false</strikeout>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton#btnAddVehicle {
background-color: #6246ff; 
  border: none;
  color: white;
  padding: 15px 32px;
  text-align: center;
  text-decoration: none;
  display: inline-block;
  font-size: 16px;
  transition-duration: 0.4s;
}

QPushButton#btnAddVehicle:hover {
  background-color: #215a91; /* Green */
  color: white;
}</string>
      </property>
      <property name="text">
       <string>Add a Vehicle</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btnAddCustomer">
      <property name="font">
       <font>
        <pointsize>-1</pointsize>
        <underline>false</underline>
        <strikeout>false</strikeout>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton#btnAddCustomer {
background-color: #6246ff; 
  border: none;
  color: white;
  padding: 15px 32px;
  text-align: center;
  text-decoration: none;
  display: inline-block;
  font-size: 16px;
  transition-duration: 0.4s;
}

QPushButton#btnAddCustomer:hover {
  background-color: #215a91; /* Green */
  color: white;
}</string>
      </property>
      <property name="text">
       <string>Add Customer</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btnOptions">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <pointsize>-1</pointsize>
        <underline>false</underline>
        <strikeout>false</strikeout>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton#btnOptions {
background-color: #6246ff; 
  border: none;
  color: white;
  padding: 15px 32px;
  text-align: center;
  text-decoration: none;
  display: inline-block;
  font-size: 16px;
  transition-duration: 0.4s;
}

QPushButton#btnOptions:hover {
  background-color: #215a91; /* Green */
  color: white;
}</string>
      </property>
      <property name="text">
       <string>Options</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="viewRentedBtn">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <pointsize>-1</pointsize>
        <underline>false</underline>
        <strikeout>false</strikeout>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton#viewRentedBtn {
background-color: #6246ff; 
  border: none;
  color: white;
  padding: 15px 32px;
  text-align: center;
  text-decoration: none;
  display: inline-block;
  font-size: 16px;
  transition-duration: 0.4s;
}

QPushButton#viewRentedBtn:hover {
  background-color: #215a91; /* Green */
  color: white;
}</string>
      </property>
      <property name="text">
       <string>View Rented Cars</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <zorder>layoutWidget</zorder>
  <zorder>layoutWidget</zorder>
  <zorder>lblBackgroundTitle</zorder>
  <zorder>layoutWidget</zorder>
  <zorder>btnRent</zorder>
  <zorder>groupBox</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
